peep.py

```diff
@@ -920,7 +920,7 @@ def peep_port(paths):
         if not hashes:
             print(req.req)
         else:
-            print('%s' % req.req, end='')
+            print('%s' % (req.link if getattr(req, 'link', False) else req.req), end='')
```

Comment:
Whilst it doesn't affect the end result, I think `getattr(req, 'link', None)` would be more semantically valid? (ie `None` instead of `False`)